### PR TITLE
build structure, layout, routes

### PR DESCRIPTION
**Pull request by baovietkentech**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,11 +34,11 @@
     "@types/node": "^14.14.41",
```
Comment (tugi93) [inline package.json:34]: @baovietkentech do you need @types/lodash?

**Comment by tienpm98** [comment]
@baovietkentech you can add "historyApiFallback: true" in webpack.dev.js file. I think it can solve your router problem
```
devServer: {
  historyApiFallback: true;
}
```

File: src/components/common/NavbarDemo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { CartType } from "../../../features/CartDemo/CartSlice";
+import React from "react";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+import { RootState } from "store";
+
+function NavbarDemo() {
+  const { amount, cartItem } = useSelector((state: RootState) => state.cart);
+
+  console.log(cartItem);
```
Comment (tugi93) [inline src/components/common/NavbarDemo/index.tsx:10]: @baovietkentech forget to remove it?

File: src/models/cart/index.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export interface CartItem {
+  id: string;
```
Comment (tugi93) [inline src/models/cart/index.ts:2]: @baovietkentech why do we need CartIte as a model?

**Review by tugi93 — COMMENTED** [review]
All in all good 👍

File: package.json (hunk 1 above)
Comment (baovietkentech) [reply package.json:33]: No, I removed it.

File: src/components/common/NavbarDemo/index.tsx (hunk 2 above)
Comment (baovietkentech) [reply src/components/common/NavbarDemo/index.tsx]: Removed it

File: src/models/cart/index.ts (hunk 3 above)
Comment (baovietkentech) [reply src/models/cart/index.ts]: Removed it

**Pull request merged**